### ISSUE TITLE
Make the login/sign up page more friendly

### ISSUE THREAD

**Issue by thebitguru** [CV19Assist/app#91]
The current login page is used as both a sign up and a login page.  This can be confusing for new users.  We need to break this up into two sections (or steps/pages), one for login and one for sign up. Ideally, the sign up page would account for the volunteer information as well.

**Comment by tescher** [comment]
@thebitguru were you still going to do a mockup or do you want me to do one?

**Comment by thebitguru** [comment]
It would be great if you can help with a very rough mock up before starting to code.  We need to highlight the sign-up workflow even if it's very similar to the login workflow.  By the default the login page would show and have a `Don't have an account? Sign up` at the bottom which launches the sign up workflow described below.

For sign up, I was thinking of using [the stepper component](https://material-ui.com/components/steppers/) to create an [experience like this](https://www.joomshaper.com/forums/multistep-form).  The first step in our case would be `Account Creation` followed by `Volunteer Profile` that uses the form on `NewUserPage`.

**Comment by tescher** [comment]
@thebitguru Couple questions
- Are you still going to allow Google login as an option?
- If so, when they log in and we find they are missing one of the required profile items (FN, LN. Email, Phone), should we immediately take them to that page to fill those in?

**Comment by thebitguru** [comment]
Yes, Google login will still be supported.  Over time we will also add Twitter and Facebook logins.

Yes, we should require the profiles for users logging in.
